mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,75 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Enlarge Image Styles
+   */
+  enlargeClassName?: string;
+
+  /**
+   * Enlarge Image Styles
+   */
+  isDisableZoom?: boolean;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
-  const className = clsx(styles.image, props.className);
+  const {src, alt, dataCy, className, enlargeClassName, isZoomable, isDisableZoom} = props;
```

Comment:
get rid of params destructuring here, please)

Reply:
i tried to change it here, but i'm not very sure what's wrong

Reply:
described in the video below